### PR TITLE
ISIC 2017 Improved Unet segmentation, Tord Skajaa Gunnarsli, s4823030

### PR DESCRIPTION
**Pull request by Tordsg**
This is my implementation of the UNET Segmentation ISIC 2017. I achieved a Dice Score of 0.81.

I followed the paper referenced by the project description. 

Training has mostly been done in Google Colab, as the Rangpur cluster has long queue issues.

For details of result, output images and project, please view the README.md

Thanks,
Tord Skajaa Gunnarsli
s4823030

### CODE REVIEW

**Comment by wangzhaomxy** [comment]
This is an initial inspection, no action is required at this point.

- Well organized files.
- Algorithm solves the problem appropriately. Average Dice Coefficient in testing dataset: 0.8104. Predicted Mask output looks good.
- Correctly using PyTorch to construct the Improved U-Net models and functions.
- Good coding design.
- Basic comment in the code, no docstring, also need some references.
- Easy difficulty problem, no higher difficulty marks.

**Comment by shakes76** [comment]
# Marking
## Good Practice (Design/Commenting, TF/Torch Usage)
Adequate design and implementation
Good spacing and comments
Header blocks missing -1
## Recognition Problem 
Solves problem 
Driver Script present
File structure present
Shows Usage & Demo & Visualisation & Data usage
Module present
Commenting minimal -1
No Data leakage
Difficulty: Easy -10
## Commit Log
Meaningful commit messages could be better -1
Progressive commits used
## Documentation
ReadMe acceptable, no refs, no usage -2
Model/technical explanation
Good Description and Comments 
Markdown used and PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Feedback required
Request Description good

**Comment by shakes76** [comment]
Please remove the gitignore file so I can merge, does not affect grade only the merge.

**Comment by Tordsg** [comment]
gitignore has been removed.

**Comment by wangzhaomxy** [comment]
Feedback attempt and no feedback marks lost.
